.github/CONTRIBUTING.md

```diff
@@ -119,27 +119,56 @@ yarn run bootstrap
 
 ### Testing
```

Comment:
Update CONTRIBUTING document.

The document descripbe following tests.

- Unit Test – Run [packages/*](../packages/) tests
- Example Test – Run [examples/*](../examples/) as test
- Integration Test – Run [Integration test](../test/integration-test)